aggregators/1.0/modules/ROOT/pages/aggregators-studio-configuration.adoc

```diff
@@ -0,0 +1,148 @@
+This configuration option also works for non-group aggregators, such as the *Time based aggregator* and *Size based aggregator* scopes.
+
+
+[[add-connector-operation]]
+== Add an Aggregator Scope to the Flow
+
+When you add a module scope to your flow, you are specifying an action for that module to perform.
+
+To add a scope for Aggregators Module, follow these steps:
+
+. In the *Mule Palette* view, select *Aggregators* and then select the desired scope:
++
+* *Group based aggregator* +
+Enables you to aggregate elements into groups by group ID.
+* *Size based aggregator* +
+Enables you to aggregate elements until a predefined size number of elements completes the aggregation.
+* *Time based aggregator* +
+Enables you to aggregate elements within a predefined time limit.
++
+. Drag the scope onto the Studio canvas to the right of the source.
+
+.Aggregator Module scopes in Studio
+image::aggregators-scopes.png[Aggregator Module scopes and source in the Mule Palette view of Anypoint Studio]
+
+[[configure-os-aggregator]]
+== Configure an Object Store for an Aggregator
+
+For any aggregator scope, you can configure an object store either by referencing a global object store or creating a private one.
+
+=== Configure a Global Object Store
+
+To configure a global object store in Studio, follow these steps:
+
+. Select the aggregator scope from the flow.
+. On the aggregator scope configuration screen, click *Advanced*.
+. Click the green plus sign (+) to create a global object store configuration.
+. In the *Global Object store configuration* screen, set *Name* to `aGlobalObjectStore`.
+. Complete the required configuration attributes and click *OK*.
+
+The following screenshot shows the *Advanced* tab of *Group based aggregator* scope with an object store configuration:
+
+image::aggregators-os-global.png[The Object Store(*) field has the aGlobalObjectStore configuration selected]
+
+In the *Configuration XML* view, the `objectStore` configuration looks like this:
+
+[source,xml,linenums]
+----
+<aggregators:group-based-aggregator name="globalOSAggregator"
+                                   maxSize="10"
+                                   objectStore="aGlobalObjectStore">
+
+----
+
+=== Configure a Private Object Store
+
+To configure a private object store in Studio, you need to manually update the XML code:
+
+. Select the aggregator scope from the flow.
+. Click *Configuration XML* in your Studio canvas.
+. Add the XML private object store `<os:private-object-store>` inside the `<aggregators:object-store>` section, for example:
+
+[source,xml,linenums]
+----
+<aggregators:size-based-aggregator  name="privateOSAggregator" maxSize="10">
+    ...
+    <aggregators:object-store>
+        <os:private-object-store alias="privateObjectStore" persistent="false"/>
+    </aggregators:object-store>
+</aggregators:group-based-aggregator>
+----
+
+[[view-app-log]]
+== View the App Log
+
+To check for problems, you can view the app log as follows:
```

Comment:
Although checking for problems is what they are doing, I wonder if we should avoid implying that problems will undoubtedly occur or are likely?
How about...:
```suggestion
To monitor app performance, view the app log as follows:
```

Reply:
If this ^ is part of the template, let's discuss at a team meeting.